docs/README.md

```diff
@@ -1,112 +1,132 @@
-# How proxless works
-  
-> Proxless looks for any kubernetes services with specific annotations and make their associated deployment serverless.
- 
-It works with 2 modes:
+# How works proxless
+
+## Wording
+
+- `service` == `kubernetes service`
+- `deployment` == `kubernetes deployment`
+
+## In 1 minute
+
+The main idea when building proxless is to be as simple as possible.  
+You don't wanna install a Custom Resource Definition, you don't want to have a big stack, you just want something to scale up and down your deployment based on-demand.  
+Proxless is a simple proxy written in golang and does not consume much resources. You don't need anything else.
+
+You just need to add an annotation on the services you want to make on-demand and Proxless will handle the rest.
```

Comment:
comment - in the future, may want to try to rewrite "you" => actions "The only requirement  for proxless is to add annotations to services ..."